File: cpp/2022/prelim/p02.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    double weight, height;
    cin >> weight >> height;
    const double bmi = weight / height / height;
    cout << fixed << setprecision(2) << bmi << endl;
    if (bmi < 20) cout << "Underweight";
    else if (bmi < 25) cout << "Healthy Weight";
    else if (bmi < 30) cout << "Overweight";
    else cout << "Obesity";
}

File: java-2022-prelim/P02.java
import java.util.Scanner;

public class P02 {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double weight = scanner.nextDouble();
        double height = scanner.nextDouble();
        double bmi = weight / height / height;
        System.out.printf("%.2f%n", bmi);
        if (bmi < 20) System.out.println("Underweight");
        else if (bmi < 25) System.out.println("Healthy Weight");
        else if (bmi < 30) System.out.println("Overweight");
        else System.out.println("Obesity");
    }
}
